lib/rubocop/cop/mixin/space_after_punctuation.rb

```diff
@@ -20,7 +20,7 @@ def investigate(processed_source)
 
       def space_forbidden_before_rcurly?
         cfg = config.for_cop('Style/SpaceInsideBlockBraces')
-        style = cfg['Enabled'] ? cfg['EnforcedStyle'] : 'space'
+        style = cfg['EnforcedStyle'] ? cfg['EnforcedStyle'] : 'space'
```

Comment:
I haven't worked out the logic of this PR yet, but just regarding code style, this should be `style = cfg['EnforcedStyle'] || 'space'`. Likewise for the others below.